### PR TITLE
Chores/client data refresh additional tests

### PR DESCRIPTION
**Pull request by ianliuwk1019**
Additional tests for CLIENT Data Refresh service.
Additionally configure Jest setting for "resetMocks" and "restoreMocks".

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
 - [api](https://fom-2.apps.silver.devops.gov.bc.ca/api)
 - [admin](https://fom-2.apps.silver.devops.gov.bc.ca/admin)
 - [public](https://fom-2.apps.silver.devops.gov.bc.ca/public)

Once merged, code will be promoted and handed off to following workflow run.
 - [Main Merge Workflow](https://github.com/bcgov/nr-fom/actions/workflows/merge-main.yml)

### CODE REVIEW

File: api/jest.config.js [hunk 1]
```diff
@@ -27,5 +27,7 @@ module.exports = {
     "cobertura"
   ],
   testResultsProcessor: "jest-sonar-reporter",
-  testEnvironment: "node"
+  testEnvironment: "node",
+  resetMocks: true,
+  restoreMocks: true
```
Comment (MCatherine1994) [inline api/jest.config.js:36]: Hi Ian, just wondering if we want to add some comment here about what they are doing and why we need them? thanks!!

Comment (ianliuwk1019) [reply api/jest.config.js:36]: Good idea, thanks Catherine, Information added.

**Review by MCatherine1994 — APPROVED** [review]

**Comment by MCatherine1994** [comment]
Hi Ian, if you could merge main into this branch, otherwise looks good, thanks!!

**Comment by ianliuwk1019** [comment]
> Hi Ian, if you could merge main into this branch, otherwise looks good, thanks!!

Thanks Catherine! Updated.

**Pull request merged**